scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -65,43 +59,40 @@
     ]
 )
 
-# Support exception for latest central and sensor 3.74 as per
+# Currently there are no support exceptions, the last one expired on 2024-06-30, see:
 # https://issues.redhat.com/browse/ROX-18223
```

Comment:
Has this been confirmed with Marina? The expiration is only 5 days old, maybe there are some talks to extend it further?

Reply:
I'll ask her 👍

Reply:
She's on vacation but will be back to work on Monday according to her Slack status.